src/dev/display/ili9341/lv_ili9341.c

```diff
@@ -0,0 +1,118 @@
+/**
+ * @file lv_ili9341.c
+ *
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+#include "lv_ili9341.h"
+
```

Comment:
Please add `LV_USE_ILI9341` guard here and and also in `lv_conf_template.h` (scripts/lv_conf_internal_gen.py needs to run as well)